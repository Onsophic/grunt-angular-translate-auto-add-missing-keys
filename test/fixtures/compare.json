{
  "entity_course": "cursus",

  "entity_learningProgram": "leerprogramma",
  "entity_learningPrograms": "leerprogramma's",

  "entity_courses": "cursussen",
  "entity_modules": "modules",

  "entity_chapter": "hoofdstuk"
}