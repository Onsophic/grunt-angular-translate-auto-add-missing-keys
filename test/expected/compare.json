{
  "entity_learningProgram": "leerprogramma",
  "entity_learningPrograms": "leerprogramma's",
  "entity_course": "cursus",
  "entity_courses": "cursussen",
  "entity_module": "$$$ module",
  "entity_chapter": "hoofdstuk"
}